The following modules were not unloaded:
  (Use "module --force purge" to unload all):

  1) xalt/2.10.45
lmod: xalt was automatically unloaded
Traceback (most recent call last):
  File "alcluster.py", line 5, in <module>
    from pyscf.geomopt.berny_solver import optimize
  File "/home/x-wmiao/.conda/envs/n2v_envi/lib/python3.8/site-packages/pyscf/geomopt/berny_solver.py", line 28, in <module>
    from berny import Berny, geomlib, coords
ModuleNotFoundError: No module named 'berny'
srun: error: a114: task 0: Exited with exit code 1
Command exited with non-zero status 1
	Command being timed: "srun --cpu-bind=cores python -u alcluster.py"
	User time (seconds): 0.00
	System time (seconds): 0.00
	Percent of CPU this job got: 1%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:01.10
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 8364
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 1254
	Voluntary context switches: 31
	Involuntary context switches: 2
	Swaps: 0
	File system inputs: 0
	File system outputs: 0
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 1
